production.ini

```diff
@@ -0,0 +1,55 @@
+[app:main]
+use = egg:network_manager
+pyramid.reload_templates = true
+pyramid.includes =
+    pyramid_tm
+
+POSTGRES_DATABASE_URL = postgresql+psycopg2://:@localhost:5432/noderegistry
+SQLITE_DATABASE_URL = sqlite:///%(here)s/sqltutorial.sqlite
+
+DATABASE_CONNECTION_URL = POSTGRES_DATABASE_URL
+
+sqlalchemy.url = %(DATABASE_CONNECTION_URL)s
+
+[server:main]
+use = egg:waitress#main
+listen = localhost:6543
+
+[alembic]
+alembic -c network_manager/alembic/alembic.ini upgrade head
```

Comment:
Are you sure this runs the upgrade command? If it does, the path to the alembic.ini should be updated to the correct location

Reply:
Line removed